blueprints/woo-shipping/blueprint.json

```
@@ -6,25 +6,14 @@
 		"author": "calvinrodrigues500",
 		"categories": ["woocommerce", "shipping", "flat_rate"]
 	},
+	"plugins":["woocommerce"],
+	"landingPage": "/wp-admin/edit.php?post_type=product",
```

```suggestion
	"landingPage": "/wp-admin/admin.php?page=wc-settings&tab=shipping",
```

Would it make more sense to open shipping settings instead of the product list?